src/include/distributed/deparser.h

```diff
@@ -121,6 +121,29 @@ extern void AppendGrantedByInGrant(StringInfo buf, GrantStmt *stmt);
 extern void AppendGrantSharedPrefix(StringInfo buf, GrantStmt *stmt);
 extern void AppendGrantSharedSuffix(StringInfo buf, GrantStmt *stmt);
 
+/* Common deparser utils */
+
+struct option_format
+{
+	const char *name;
+	const char *format;
+	const int type;
+};
```

Comment:
* We usually wrap structs with `typedef`.
* I don't think that fields need to be constant but maybe the object itself should be declared with `const` flag.
* Usually we don't use snake-case in function / struct names.
* Let's be more clear about what option it is (DefElemOptionFormat)

```suggestion
typedef struct DefElemOptionFormat
{
	char *name;
	char *format;
	int type;
} DefElemOptionFormat;
```